frontend/src/lib/lemon-ui/LemonTable/types.ts

```diff
@@ -20,6 +20,8 @@ export interface LemonTableColumn<T extends Record<string, any>, D extends keyof
     /** Tooltip to display on title hover. An info icon ("i" in circle) is shown when a tooltip is available. */
     tooltip?: string
     key?: string
+    /** If true, the column is not displayed. Optional, defaults to not disabled. */
+    is_disabled?: boolean
```

Comment:
maybe `is_visible` is better suited here - we use `disabled` elsewhere for when a UI element is still visible but not interactive

Reply:
That makes sense, I do want to keep it as a negative for semantic and optional parameterization reasons, so I'll switch to `isHidden`.